Packages/Tracking/Core/Editor/Scripts/BuildValidation.cs

```diff
@@ -18,18 +25,33 @@ public void OnPreprocessBuild(BuildReport report)
 
     private void AndroidBuild()
     {
+        if (!UltraleapSettings.Instance.showAndroidBuildArchitectureWarning)
+            return;
+
         if (PlayerSettings.Android.targetArchitectures != AndroidArchitecture.ARM64) // Must be only arm64 so direct equality check, not checking for IL2CPP explicitly because it's required to select ARM64 anyway...
+        {
             DoValidationPopup($"Target architecture for an Android build including Ultraleap hand tracking must be ARM64.{Environment.NewLine}" +
                               $"To fix this, go to 'Edit -> Project Settings -> Player -> Other Settings -> Configuration'.{Environment.NewLine}" +
-                              "Set 'Scripting Backend' to 'IL2CPP' and 'Target Architectures' to only 'ARM64'.");
+                              "Set 'Scripting Backend' to 'IL2CPP' and 'Target Architectures' to only 'ARM64'.", ValidationPopupType.ANDROID_ARCHITECTURE);
+        }
     }
 
-    private void DoValidationPopup(string error)
+    private void DoValidationPopup(string error, ValidationPopupType validationType)
     {
         if (EditorUtility.DisplayDialog("Ultraleap build validation",
             $"Validation issue encountered: {error} Do you want to continue with the build anyway?", "Continue Building",
             "Stop Build"))
+        {
+            switch(validationType)
+            {
+                case ValidationPopupType.ANDROID_ARCHITECTURE:
+                    UltraleapSettings.Instance.showAndroidBuildArchitectureWarning = false;
+                    UnityEngine.Debug.LogWarning("Ultraleap Android architecture warning has been muted. To unmute, re-enable in the Ultraleap Settings tab in the Project panel");
```

Comment:
This logic might be the wrong way round, the warning should be logged if you choose to continue.

Reply:
Being inside the "if" of the DisplayDialog, it should _only_ be called when you select continue. When selecting "Stop Build", the if DisplayDialog returns false